v2/passwordless/serverless/with-aws-lambda/api-gateway-config.mdx

```diff
@@ -1,43 +0,0 @@
----
```

Comment:
do not remove this file, instead use the `<Redirector ...>` component here which redirects the user to the right page. You do not need to inlcude this in the sidebar.json. This is only done cause google might have linked to this file and then we don't want to give a 404 to the user.

Reply:
done